src/lib/journey/callbacks/select-idp/select-idp.stories.js

```diff
@@ -1,15 +1,21 @@
 import { FRStep, CallbackType } from '@forgerock/javascript-sdk';
 
 import {
-  socialStep,
-  socialStepLocalAuthNoForm,
-  socialStepLocalAuthWithForm,
+  multipleProvidersLocalAuthFormStep,
+  multipleProvidersLocalAuthNoFormStep,
+  multipleProvidersNoLocalAuthStep,
+  singleProviderLocalAuthFormStep,
+  singleProviderLocalAuthNoFormStep,
+  singleProviderNoLocalAuthStep,
 } from './select-idp.mock';
 import Input from './select-idp.story.svelte';
 
-const social = new FRStep(socialStep);
-const socialLocalAuthNoForm = new FRStep(socialStepLocalAuthNoForm);
-const socialLocalAuthWithFormStep = new FRStep(socialStepLocalAuthWithForm);
+const multipleProvidersLocalAuthForm = new FRStep(multipleProvidersLocalAuthFormStep);
+const multipleProvidersLocalAuthNoForm = new FRStep(multipleProvidersLocalAuthNoFormStep);
+const multipleProvidersNoLocalAuth = new FRStep(multipleProvidersNoLocalAuthStep);
+const singleProviderLocalAuthForm = new FRStep(singleProviderLocalAuthFormStep);
+const singleProviderLocalAuthNoForm = new FRStep(singleProviderLocalAuthNoFormStep);
+const singleProviderNoLocalAuth = new FRStep(singleProviderNoLocalAuthStep);
```

Comment:
I felt it was beneficial to test the scenarios around local auth with single or multiple IDPs now that we are supporting more than one. If there are any I've left out let me know.